docs/_static/auth/iam.yaml

```diff
@@ -819,6 +823,14 @@ components:
         user_role:
           type: string
           description: Role of the End-User.
+    WalletOwnerType:
+      type: string
+      description: |
+        Wallet owner type that should fulfill the presentation definition.
+        Can either be an organization wallet or a user (personal) wallet.
+      enum:
+        - organization
+        - user
```

Comment:
A machine can also be a user. I would stick to `person` since personal wallet is the common name for this type.

Reply:
I agree; "user" is a technical term for the actual user of the Nuts node, while what was intended is the sort of entity (legal organization v.s. natural person) that holds the wallet. But since it's already used in other places, I'll make an  issue for it.